frontend/lib/src/components/widgets/CameraInput/CameraInput.test.tsx

```diff
@@ -132,87 +112,7 @@ describe("CameraInput widget", () => {
         value: LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED,
       },
     })
-    const wrapper = mount(<CameraInput {...props} />)
-    expect(wrapper.find("StyledWidgetLabel").prop("labelVisibility")).toEqual(
-      LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED
-    )
-  })
-
-  it("shows a SwitchFacingMode button", () => {
-    const props = getProps()
-    const wrapper = mount(<CameraInput {...props} />)
-
-    act(() => {
-      wrapper
-        .find("Webcam")
-        .props()
-        // @ts-expect-error
-        .onUserMedia(null)
-    })
-
-    wrapper.update()
-
-    expect(wrapper.find("SwitchFacingModeButton").exists()).toBeTruthy()
-  })
-
-  it("changes `facingMode` when SwitchFacingMode button clicked", () => {
-    const props = getProps()
-    const wrapper = mount<CameraInput, Props, State>(
-      <CameraInput {...props} />
-    )
-
-    act(() => {
-      wrapper
-        .find("Webcam")
-        .props()
-        // @ts-expect-error
-        .onUserMedia(null)
-    })
-
-    wrapper.update()
-
-    act(() => {
-      wrapper.find("SwitchFacingModeButton").find("button").simulate("click")
-    })
-    wrapper.update()
-
-    expect(wrapper.instance().state.facingMode).toBe(FacingMode.ENVIRONMENT)
-  })
-
-  it("test handle capture function", async () => {
-    const props = getProps()
-    const wrapper = shallow<CameraInput, Props, State>(
-      <CameraInput {...props} />
-    )
-    // @ts-expect-error
-    await wrapper.instance().handleCapture("test img")
-
-    expect(wrapper.instance().state.files).toHaveLength(1)
-    expect(wrapper.instance().state.files[0].name).toContain("camera-input-")
-    expect(wrapper.instance().state.shutter).toBe(false)
-    expect(wrapper.instance().state.minShutterEffectPassed).toBe(true)
-
-    expect(wrapper.find(StyledBox)).toHaveLength(1)
-    expect(wrapper.find(WebcamComponent)).toHaveLength(0)
-  })
-
-  it("test remove capture", async () => {
```

Comment:
Couldn't really replicate with RTL -> `handleCapture` func triggered by **Take Photo** button passes `undefined`...

Deleted since E2E test coverage includes Take Photo and Clear Photo functionality